### ISSUE TITLE
Use GitHub Actions to build documentation

### ISSUE THREAD

**Issue by bquorning** [rubocop/docs.rubocop.org#24]
Pulling repositories and building documentation takes 10–13 minutes on my very fast laptop, and I don’t know if there’s any way to speed it up. So I was thinking if we could at least move the work to GitHub Actions, meaning that I would only need to update the playbook file and then GHA would do the actual work – is that possible?

**Comment by bquorning** [comment]
cc @bbatsov

**Comment by bbatsov** [comment]
Yeah, it's possible, I just never got to doing it - https://docs.antora.org/antora/latest/publish-to-github-pages/#using-github-actions

**Comment by bquorning** [comment]
It looks like you’re currently working on this issue @bbatsov 🎉

**Comment by bbatsov** [comment]
Oh, I had forgotten we had an issue about this. :D Yeah, I finally told myself that enough is enough and this needs to be properly automated. 

The new job seems to be working properly, so I'll close this ticket.

**Comment by bquorning** [comment]
I am not sure it works actually. I'm seeing at least some dead links on https://docs.rubocop.org/ but I can't have a closer look just now.

**Comment by bquorning** [comment]
Also the documented Docker command doesn't work.

**Comment by bbatsov** [comment]
> Also the documented Docker command doesn't work.

It was submitted by someone else, so I haven't tried it.

> I am not sure it works actually. I'm seeing at least some dead links on https://docs.rubocop.org/ but I can't have a closer look just now.

I think that's something to do with the use of `latest` that I attempted. I'm puzzled why this doesn't work. (see https://docs.antora.org/antora/latest/playbook/urls-latest-version-segment-strategy/) I'll change it back to a hardcoded version and I think it will fix the index.

**Comment by bquorning** [comment]
You can leave the Docker command to me, I'll try to fix it.
